scripts/variantstore/tieout/add_max_as_vqslod.py

```diff
@@ -19,33 +19,30 @@
 
         parts = line.split("\t")
         
-        if ("ExcessHet" in parts[6]):
+        # strip out hard filtered sites, so vcfeval can use "all-records" to plot the ROC curves
```

Comment:
probably just because I am too new here, but seeing an example in the comments would be really helpful, esp for the transformation that starts on line 37 where we seem to be putting "new" data into parts[6]?